terraform/deployments/search-api-v2/datastore_schema.tf

```diff
@@ -0,0 +1,203 @@
+resource "restapi_object" "google_discovery_engine_datastore_schema" {
+  path      = "/dataStores/${google_discovery_engine_data_store.govuk_content.data_store_id}/schemas"
+  object_id = "default_schema"
+
+  data = jsonencode({
+    structSchema = {
+      "$schema" = "https://json-schema.org/draft/2020-12/schema"
+      type      = "object"
+      properties = {
+        # Unique content ID from the Publishing API (for debugging/retrieving single objects)
+        content_id = {
+          type        = "string"
+          retrievable = true
+        }
+        # Unique identifier for search quality evaluation (identical to content_id)
+        #
+        # Note: This is only used to compare results for evaluation purposes, and is not a URL
+        # despite the name. We decided against using the alternative 'uri' field name to avoid
+        # confusion with our existing 'url' field and 'uri' key property (see
+        # https://cloud.google.com/generative-ai-app-builder/docs/evaluate-search-quality)
```

Comment:
Why have the descriptions been added as comments rather than in a description field as they were previously? Is the description on the fields not supported by VAIS?

Reply:
Correct – `description` is not supported by VAIS, and even though it's currently silently ignored (unlike some other fields such as `required` which now seem to do _something_ under the hood), I've removed them and turned them into comments.

The nice thing about this being HCL/Terraform now is that we can actually have comments (Terraform's JSON parser didn't support the unofficial JSON-with-comments format so we couldn't with the previous setup) ✨